assets/redpajama.yaml

```diff
@@ -0,0 +1,24 @@
+---
+
+- type: dataset
+  name: RedPajama-Data
+  organization: Together
+  description: The RedPajama base dataset is a 1.2 trillion token fully-open dataset
+    created by following the recipe described in the LLaMA paper
+  created_date: 2022-04-17
+  url: https://www.together.xyz/blog/redpajama
+  datasheet: https://huggingface.co/datasets/togethercomputer/RedPajama-Data-1T
+  modality: text
+  size: 1.2 trillion tokens
+  sample: []
+  analysis: ''
+  dependencies: [LLaMA dataset, GitHub data, Wikipedia dataset]
```

Comment:
you can remove "data" for GitHub and Wikipedia. You only need to say that when describing something like "LLaMA dataset" which means "the dataset used to train LLaMA". GitHub and Wikipedia are not models but data already.

This is also important for making sure the graph is correct. We want to make sure the dependencies use the exact same strings, since otherwise stuff won't get connected as we intend.